### PR TITLE
pbTests: Update/Fix vagrant.yml playbook

### PR DESCRIPTION
**Pull request by Willsparker**
A few changes to make this better:
- Add the gather_facts output
- Remove unnecessary checks to the folders that are being created. Ansible's `file` module will check and skip creating the folder if it's already there.
- Install jdk8 using APIv3
- Rework the tasks that symlink `java` and `javac` to the `/usr/bin/` folder - Ansible's `file` module does not allow for wildcard searches, so I had to use `find` and register what was in the `/usr/lib/jvm` folder.
- Add `tags` to everything
- Fix typo in QEMU apt list
- Install QEMU 5.0.0 instead of 4.2.0 - I'm unable to `ssh` to the Fedora Rawhide RISC-V QEMU box in QEMU `4.2.0`, but it's fine in `5.0.0`

### CODE REVIEW

**Comment by Willsparker** [comment]
Ref: The linter.
I can't see a trailing space on L.98 - could it be referring to the spaces before the comment on line 99? :thinking:

File: ansible/playbooks/vagrant.yml [hunk 1]
```diff
@@ -35,63 +51,62 @@
...
+          find:
+            paths: /usr/lib/jvm
+            file_type: directory
+            contains: jdk8u*
+          register: jdk8_folder
           when: default_java.rc != 0
-
-        - name: Assign JDK-8 as default javac
+          tags: java
+
```
Comment (sxa) [inline ansible/playbooks/vagrant.yml:98]:
```suggestion

```

**Review by sxa — CHANGES_REQUESTED** [review]
Not sure why you can't see it but there is very definitely one space character on the "blank" linke 98

**Comment by Willsparker** [comment]
hmm, weird. Thanks :-)

**Comment by Haroon-Khel** [comment]
Currently testing on a x64 ubuntu 18 machine

**Review by Haroon-Khel — APPROVED** [review]
LGTM. Playbook ran fine on a ubuntu 18 x64 machine. The Qemu part installed without error and worked as expected with the qemuPlaybookCheck.sh script

**Review by sxa — APPROVED** [review]
Approved, but I would suggest holding off on merging until we've got it gully running with VPC/QPC via jenkins ;-)

**Comment by karianna** [comment]
Safe to merge now?

**Comment by sxa** [comment]
@karianna No - we're replacing the machine that this stuff is running on and the new one hasn't been tested yet (i.e. hasn't had VPC/QPC run on it via jenkins). Should hopefully be in the next day or so as Will is moving stuff across for me as we speak

**Comment by Willsparker** [comment]
Managed to get a QPC run on Jenkins - https://ci.adoptopenjdk.net/job/QEMUPlaybookCheck/138/ARCHITECTURE=s390x,OS=ubuntu18,label=vagrantX/consoleFull
I'll do a VPC run once my current one has finished :-)

**Comment by Willsparker** [comment]
https://ci.adoptopenjdk.net/job/VagrantPlaybookCheck/887/
`VPC` run to check the playbook does it's job :+1:

**Comment by Willsparker** [comment]
Ref: Latest (2) commit(s)
`qemuPlaybookCheck.sh` couldn't run all of the platforms due to some packages missing, which have now been added in :-) also rearranged the QEMU package list in alphabetical order

**Comment by Willsparker** [comment]
https://ci.adoptopenjdk.net/job/QEMUPlaybookCheck/140/
Looks good now! `riscv` failed as it relys on #1483 , Deb10/aarch64 seems to have failed due to a compiler reason- however it still was able to start the run, so it shows the machine is setup correctly :-)

**Pull request merged**